spec/support/request/shop_workflow.rb

```diff
@@ -16,9 +16,9 @@ def set_order(order)
     ApplicationController.any_instance.stub(:session).and_return({order_id: order.id, access_token: order.token})
   end
 
-  def add_product_to_cart
```

Comment:
How did this actually work before? How did this method have access to `order` and `product`? Is it because `let` defines a method instead of a variable?